grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -39,10 +43,12 @@ class WpsService extends AsyncDownloadService {
     }
 
     def notifyViaEmail(params) {
+
+        params.expirationPeriod = _getExpirationPeriod()
```

Comment:
I didnt try the tag in the GSP. I didnt even know about it. However the simple pretty time format function didnt give me an accurate enough time which i assume the tag would do. 

The `_getExpirationPeriod` function is now quite bullet proof which im happy about. I'd also rather have it so config can be forgotten or left off by us or others.